src/applications/search/containers/SearchApp.jsx

```diff
@@ -57,6 +57,9 @@ class SearchApp extends React.Component {
     // If there's data in userInput, it must have come from the address bar, so we immediately hit the API.
     const { userInput, page } = this.state;
     if (userInput) {
+      if (userInput.length >= 255) {
```

Comment:
**Question:** The description says if the query is longer than 255 characters, it should show the error message. But this logic is if the query is as long as, or longer, than 255 characters. Should it be more like `userInput.length > 255`?

Reply:
Thanks for the comment Max! My fault on the description, I will update it. It should say max allowable is 255 characters

Reply:
I think we may have some wires crossed here; character comparison should be `> 255`; Search.gov allows 255 characters, but not 256.